### ISSUE TITLE
Touchscreen + Pen issue

### ISSUE THREAD

**Issue by mfauvain** [linux-surface/linux-surface#280]
When using touchscreen AND pen (xournalpp, mypaint etc..) it always end up messing touch (touch screen not responding to touch events) after a while. Doing `modprobe -r i2c_hid` and then `modprobe i2c_hid` fixes the issue but it is a real pain.
I don't see anything in logs that would help me point in the right direction. Any help appreciated.

**Comment by SamLukeYes** [comment]
I don't think it is a bug of GNOME. The same thing happened when I used Plasma, Cinnamon, MATE, Budgie and DDE on Xorg. I haven't seen this issue since I began to use GNOME on Wayland, but I've been using pen less frequently since then, so I'm not sure whether switching to Wayland would help. My GNOME version is 3.36.6.

**Comment by mfauvain** [comment]
I use surfaceGo so touchscreen does not use ipts, however I found that disabling 'eraser' in those apps (and hence ending up not using it) helps a lot. I haven't had the problem since then... I guess it is a i2c_hid driver issue when switching from pen to eraser then...
Don't know how to investigate further though.
